crates/zk-por-cli/src/prover.rs

```diff
@@ -24,30 +33,51 @@ use zk_por_core::{
 };
 use zk_por_tracing::{init_tracing, TraceConfig};
 
+// as we use one thread to prove each batch, we load num_cpus batches to increase the parallelism.
+fn calculate_per_parse_account_num(batch_size: usize) -> usize {
+    let num_cpus = num_cpus::get();
+    let num_cpus =
+        if BATCH_PROVING_THREADS_NUM < num_cpus { BATCH_PROVING_THREADS_NUM } else { num_cpus };
+    num_cpus * batch_size
+}
+
 pub fn prove(cfg: ProverConfig, proof_output_path: PathBuf) -> Result<(), PoRError> {
-    let trace_cfg: TraceConfig = cfg.log.into();
-    let _g = init_tracing(trace_cfg);
+    let trace_cfg: TraceConfig = cfg
+        .log
+        .unwrap_or(ConfigLog {
+            file_name_prefix: "zkpor".to_string(),
+            dir: "logs/".to_string(),
+            level: "info".to_string(),
+            console: true,
+            flame: false,
+        })
+        .into();
 
-    let mut database = DataBase::new(DbOption {
-        user_map_dir: cfg.db.level_db_user_path.to_string(),
-        gmst_dir: cfg.db.level_db_gmst_path.to_string(),
-    });
+    let _g = init_tracing(trace_cfg);
 
-    let batch_size = cfg.prover.batch_size as usize;
-    let token_num = cfg.prover.num_of_tokens as usize;
+    let mut database: Box<dyn PoRDB>;
+    if let Some(level_db_config) = cfg.db {
```

Comment:
we dont use path to indicate whether enable. we should explicitly specify whether enable persistence; something like --persist in the cli.